docs/initial-setup.rst

```diff
@@ -299,6 +299,12 @@ Then in the ``testing`` and ``anotherenv`` environments, fabulaws will apply
 basic auth to the sites. For testing, user ``user1`` will be able to use password
 ``password1``, and so forth.
 
+.. NOTE::
+   Fabulaws will also turn off Basic Auth for the health check URL so that the load balancer
+   can access it. It assumes that the health check URL is ``/healthcheck.html`` and that Django will
+   be serving the health check URL (rather than being served as a static file directly by Nginx, for
+   example). If either of those assumptions are not correct, this version of Basic Auth setup may
+   not work for your situation.
```

Comment:
but you can easily tweak it by copying and modifying the template for nginx.conf.